src/human.js

```diff
@@ -69,8 +69,8 @@ module.exports = class extends EventEmitter {
     if (pack.length === 1)
       return this.pick(0)
 
-    if (this.useTimer)
-      this.time = 20 + 5 * pack.length
+    if (this.useTimer > 0)
```

Comment:
Is it apparent to the user that a value of zero is "no timer"? Maybe add some `title` text to the label text, or have both a checkbox that enables the time (and shows the timer dialog) or disables it. I might fix that myself if I get some time.